### DISCUSSION TITLE
how to develop locally on single server

### DISCUSSION THREAD

**Discussion by NkatekoDev24** [mediacms-io/mediacms#974] [General]
I have made changes on the media cms frontend scss files and  I am developing using the single server installation, how do I build the frontend files so that the changes can be served in Django?

**Comment by mgogoulos** [comment]
Have a look on [https://github.com/mediacms-io/mediacms/blob/main/docs/developers_docs.md#4-how-to-contribute , ](https://github.com/mediacms-io/mediacms/blob/main/docs/dev_exp.md) and see the commands for building and placing to static/ (even if you are not using Docker, the same commands should apply)

**Reply by NkatekoDev24** [reply]
I have followed the documents you have mentioned and executed all the commands, But I am failing to execute the last one of restarting django files. This command in particular: Restart Django - docker-compose -f docker-compose-dev.yaml restart web so that it uses the new static files, on Linux server, when I run that command it gives me this error: 

root@media-cms:/home/mediacms.io/mediacms# restart web
Command 'restart' not found, did you mean:
  command 'restartd' from deb restartd (0.2.3-1.1)
  command 'rstart' from deb x11-session-utils (7.7+4build2)
Try: apt install <deb name>

**Reply by mgogoulos** [reply]
with local setup it uses **Systemd** to handle services, so try 

```
systemctl restart web
```
